storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1445,8 +1462,9 @@ bool Redo_Log_Data_Manager::stop_at(lsn_t lsn, lsn_t checkpoint_lsn) {
   archived_log_monitor.stop();
 
   /* to ensure redo logs are not disabled during the backup, reopen the log
-  files to read HEADER */
-  if (!reopen_log_files(lsn)) {
+  files to read HEADER. */
+  if (opt_lock_ddl == false && archived_log_state == ARCHIVED_LOG_NONE &&
+      !reopen_log_files(lsn)) {
```

Comment:
if we just want to check redo header and verify the redo logs are not disabled, do we need to find the same file  with an LSN(noted at start?)?

Reply:
right. Fixed the code . now we just check if we can open redo files.